CMakeLists.txt

```diff
@@ -202,7 +202,7 @@ endif ()
 # Everything should be valid from here onwards.
 if (OPENCMISS_SETUP_TYPE STREQUAL "standard" AND NOT OPENCMISS_INDEPENDENT)
     set(OPENCMISS_MANAGE_SOURCE_DIR "${OPENCMISS_ROOT}/src/manage")
-    set(OPENCMISS_MANAGE_BINARY_DIR "${OPENCMISS_ROOT}/build/manage${SINGLE_ARCHITECUTRE_BUILD_TYPE}")
+    set(OPENCMISS_MANAGE_BINARY_DIR "${OPENCMISS_ROOT}/build/manage/${CMAKE_HOST_SYSTEM_NAME}${SINGLE_ARCHITECUTRE_BUILD_TYPE}")
```

Comment:
CMAKE_SYSTEM_NAME should be used here CMAKE_HOST_SYSTEM_NAME is for cross-compiling purposes which we are not.

Reply:
The system name should also be separated from the architecture build type.

Reply:
The system name is separated. The SINGLE_ARCHITECTURE_BUILD_TYPE has a / prepended.